src/ert/_c_wrappers/enkf/analysis_config.py

```diff
@@ -250,6 +250,8 @@ def __ne__(self, other):
         return not self == other
 
     def __repr__(self):
+        if not self._address():
```

Comment:
that sounds like a very good idea : ) thank you!

Reply:
almost feels like i should add this to `BaseCClass`, in cwrap... how do you feel about that?
and while reading the code of that class - how about making the code even less readable, and just saying
```
    if not self:
```
using the `__bool__` method of `BaseCClass` which basically checks if the c pointer is truthy? ; D

Reply:
In this case, I think `if not self._address():` is better :)

Reply:
ok - but what about getting it into cwrap? should i make an issue there? or just let it be as is?

Reply:
> ok - but what about getting it into cwrap? should i make an issue there?

Don't really think it is worth the effort it is not something critical and it will not drastically improve anything. So I guess just leave it as it is.